lib/subscriptions.js

```diff
@@ -1,71 +1,48 @@
-var subscriptions = {}
+
+var uuid = require('uuid')
 
 function Subscription(options) {
-  this.id = Math.floor(Math.random() * 1000000000).toString()
+  this.id = options.id || uuid.v4()
   this.notify_url = options.notify_url
   this.pos = options.pos || '0'
   this.filter = options.filter || {}
-  this.setTTL(options.ttl || 60 * 60 * 24)
+  this.ttl = options.ttl || 60 * 60 * 24
```

Comment:
This constant should go to top-of-file, or maybe even a config value.  I also wonder if 24hrs is too aggressive, but don't really have a frame of reference what what would be appropriate.  I guess it doesn't matter too much initially.